CppCoreGuidelines.md

```diff
@@ -10328,7 +10328,7 @@ In a class template, nonvirtual functions are only instantiated if they're used
 
 * Flag a class template that declares new (non-inherited) virtual functions.
 
-## <a name="SS-tpg-concepts"></a> TPG.concepts: Concept rules
+## <a name="SS-concepts"></a> T.concepts: Concept rules
 
```

Comment:
Ah yes, I guess the thing that confused me was that on line 1391 you changed the anchor to "SS-t-concepts"; but here it is just "SS-concepts" -- note the missing "-t".